modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -826,7 +827,12 @@ def process_draft_appointment(referral_id, referral_consult_id)
         provider = find_provider(npi: referral.provider_npi,
                                  specialty: referral.provider_specialty,
                                  address: referral.treating_facility_address)
-        return { success: false, json: provider_not_found_error, status: :not_found } unless provider&.id
+        if provider&.id.blank?
```

Comment:
Note: This is the real change here. Everything else is just cleanup.

Reply:
I think we should move CC_APPOINTMENT_ERROR_TAG to the second argument so that it becomes a named tag or some other meta data in the log. I'm not sure how those work exactly. Makes queries a bit easier.

Also can we add logging for when the provider data is empty in the case of fetching referral details?